File: ManDroid/Class1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace ManDroid
    Public Class EventRecord
        Public Property Time As String
        Public Property Type As String
        Public Property Operation As String
        Public Property Description As String

        Public Sub New(ByVal type As String, ByVal operation As String, ByVal description As String)
            Time = Date.Now.ToShortTimeString
            Operation = operation
            Type = type
            Description = description
        End Sub
    End Class

    Public Module EventType
        Public ReadOnly Info As String = "Info"
        Public ReadOnly [Error] As String = "Error"
        Public ReadOnly Complete As String = "Complete"
        Public ReadOnly Warning As String = "Warning"
        Public ReadOnly Success As String = "Success"
        Public ReadOnly Fail As String = "Fail"
        Public ReadOnly System As String = "System"
        Public ReadOnly Device As String = "Device"
    End Module
End Namespace

File: ManDroid/Class1.cs
using System;

namespace ManDroid.ManDroid
{
    public class EventRecord
    {
        public string Time { get; set; }
        public string Type { get; set; }
        public string Operation { get; set; }
        public string Description { get; set; }

        public EventRecord(string type, string operation, string description)
        {
            Time = DateTime.Now.ToShortTimeString();
            Operation = operation;
            Type = type;
            Description = description;
        }
    }

    public static class EventType
    {
        public readonly static string Info = "Info";
        public readonly static string Error = "Error";
        public readonly static string Complete = "Complete";
        public readonly static string Warning = "Warning";
        public readonly static string Success = "Success";
        public readonly static string Fail = "Fail";
        public readonly static string System = "System";
        public readonly static string Device = "Device";
    }
}

